cmd/support-diag.go

```diff
@@ -388,6 +403,20 @@ func fetchServerDiagInfo(ctx *cli.Context, client *madmin.AdminClient) (interfac
 			progressV2(info)
 		}
 		healthInfo = info
+	case madmin.HealthInfoVersion3:
```

Comment:
Why is a new version needed here? Aren't you just adding a new field?

Reply:
But if it is a nil-able struct can't it just check if it is there or not?

Reply:
If we don't introduce the new version, then with same health info version (v3), some (newer) minio versions will send the net info, whereas others won't. And mc won't know when to _expect_ this data in the response and when not to.

If this was just a part of some other top level data category (disk, os, mem, process, etc) then we did not have to show it as a separate line item in the progress, and would not have required a new version.

However, since this is a new category of data for which we want to show the progress (waiting -> received), but do it only when we know that minio is going to send this data, it is required to introduce the new version. (mc first reads only the header of the health info to get the version so that it knows what data to expect)

Reply:
Closing this PR now. Will revisit after the spinner mechanism is fixed.